ticker/src/Ticker.cpp

```diff
@@ -11,19 +11,13 @@ Ticker::~Ticker() {
 }
 
 Ticker::period_type Ticker::delta_time() const {
-    return delta_;
+    return period_;
```

Comment:
`return 1 / std::chrono::duration_cast<std::chrono::seconds>(period).count();`

Reply:
`delta_time` is a period between two ticks (i.e. two frames render), so it equals our `period_`, maybe conversion to seconds is useful here

Reply:
No, you are probably right, return the period specified, and user will decide what to do with it.